pkg/netpol/eval/check.go

```diff
@@ -350,3 +350,12 @@ func (pe *PolicyEngine) allAllowedConnections(src, dst string) (k8s.ConnectionSe
 	allowedConns, err := pe.allAllowedConnectionsBetweenPeers(srcPeer.(Peer), dstPeer.(Peer))
 	return allowedConns.(*k8sConnectionSetWrapper).ConnectionSet(), err
 }
+
+// AllowedConnectionsToPeer returns the allowed connections to a workload peer
+func (pe *PolicyEngine) AllowedConnectionsToPeer(peer Peer) Connection {
```

Comment:
rename func to `GetPeerExposedProtocolsAndPorts` (also the internal version, for a pod). 
the term "allowed connections" may be confusing